test/controllers/organizations/staff/external_form_upload_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require "test_helper"
+
+class Organizations::Staff::ExternalFormUploadControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @organization = ActsAsTenant.current_tenant
+    @file = fixture_file_upload("google_form_sample.csv", "text/csv")
+    @params = {files: [@file]}
+    @user = create(:admin, email: "adopter1@alta.com")
```

Comment:
We should be creating an `adopter` here, instead of `admin`

There are also some changes happening in this PR https://github.com/rubyforgood/pet-rescue/pull/997
- a Person can have multiple form submissions
- a form submission will be created on a person when a new adopter signs up

This means the asserstions on lines 20 and 21 will become unnecessary. We'll want to just test two cases

- when a form submission does not have any form answers (and no CSV timestamp), assert that new form answers are created
- when a form submission does have a csv timestamp matching the timestamp in the CSV, no new Form Answers are created

I think the best thing to do, here, is wait for that PR to get merged, then finish these tests off. Thoughts?

Reply:
Ok, I understand lines 20 and 21 can be removed.

On the other assertions I'm verifying that FormSubmissions and FormAnswers are being created after the upload csv. Where we have 4 answers and questions/cells and  1 form was uploaded so one FormSubmission was created. 

As I understood we may need to add a test for when a csv file matches an existing csv_timestamp entry... Nothing should happen. But all of these are subject to change after PR #997

Then I will remove the unnecessary assertions, address controller comments, and wait for your instructions on the tests.

Reply:
@wandergithub FYI this PR has now been merged https://github.com/rubyforgood/pet-rescue/pull/997
So now person has many form submissions

Reply:
@wandergithub let me know if you have any blockers on this :)

Reply:
@kasugaijin Yeah, I would like to know which requirements or specifications changed on this PR. It's not clear to me.